doc/interop-test-descriptions.md

```diff
@@ -1078,6 +1094,19 @@ included in that latency measurement (channel teardown semantics differ widely
 between languages). This latency measurement should also be the value that is
 logged and recorded in the latency histogram.
 
+Note on Concurrent Execution and Channel Creation: In a concurrent execution setting (i.e., when `soak_num_threads > 1`), each 
```

Comment:
nit: no need to special case idea of "concurrent execution".

The documentation can describe what do according to `soak_num_threads`